call the ruby implementation of epspdf(tk) explicitely with the
ruby interpreter so we don't have to make them executable
---
 texmf-dist/scripts/epspdf/epspdf   |    2 +-
 texmf-dist/scripts/epspdf/epspdftk |    4 +---
 2 files changed, 2 insertions(+), 4 deletions(-)

Index: texlive-base-2009/texmf-dist/scripts/epspdf/epspdf.rb
===================================================================
--- texlive-base-2009.orig/texmf-dist/scripts/epspdf/epspdf.rb	2010-03-07 19:12:51.000000000 +0900
+++ texlive-base-2009/texmf-dist/scripts/epspdf/epspdf.rb	2010-03-07 19:12:59.000000000 +0900
@@ -1,3 +1,3 @@
 #!/bin/sh
 script=`kpsewhich -format=texmfscripts epspdf.rb`
-"$script" "$@"
+ruby "$script" "$@"
Index: texlive-base-2009/texmf-dist/scripts/epspdf/epspdftk.tcl
===================================================================
--- texlive-base-2009.orig/texmf-dist/scripts/epspdf/epspdftk.tcl	2010-03-07 19:12:48.000000000 +0900
+++ texlive-base-2009/texmf-dist/scripts/epspdf/epspdftk.tcl	2010-03-07 19:13:10.000000000 +0900
@@ -1,7 +1,7 @@
 #!/bin/sh
 script=`kpsewhich -format=texmfscripts epspdftk.rb`
 if test "x$1" = "x"; then
-  "$script" $HOME
+  ruby "$script" $HOME
 else
-  "$script" "$@"
-fi
\ No newline at end of file
+  ruby "$script" "$@"
+fi
